omod/src/main/webapp/fragments/patientInfoPage.gsp

```diff
@@ -1189,10 +1189,10 @@ form input:focus, form select:focus, form textarea:focus, form ul.select:focus,
     </div>
 </form>
 
-<div id="printDiv" style="display: none;">
+<div id="printDiv" style="display: none;" size="50">
     <center>
         <center>
-            <img width="60" height="60" align="center" title="OpenMRS" alt="OpenMRS"
+            <img width="60" height="60" align="center" title="..." alt="..."
```

Comment:
Not sure what those .... means, they are supposed to be something like **Integrated KenyaEMR**